velox/duckdb/conversion/DuckParser.cpp

```diff
@@ -151,10 +152,16 @@ std::shared_ptr<const core::IExpr> parseFunctionExpr(ParsedExpression& expr) {
   for (const auto& c : functionExpr.children) {
     params.emplace_back(parseExpr(*c));
   }
-  return callExpr(
-      normalizeFuncName(functionExpr.function_name),
-      std::move(params),
-      getAlias(expr));
+  auto func = normalizeFuncName(functionExpr.function_name);
+  // NOT LIKE function needs special handling as it maps to two functions "not",
```

Comment:
Would you extract this change into a separate PR and add a test to DuckParserTest.cpp?

Reply:
https://github.com/facebookincubator/velox/pull/1473